Microsoft.Toolkit.Uwp.UI/Triggers/NotEqualStateTrigger.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Triggers
+{
+    /// <summary>
+    /// Enables a state if the value is not equal to another value
+    /// </summary>
+    public class NotEqualStateTrigger : StateTriggerBase
```

Comment:
Shouldn't "equal" trigger names be consistent?
NotEqualStateTrigger 
Equal**s**StateTrigger
Note ^ on 's' character. It exist in 'EqualsStateTrigger', but not in 'NotEqualStateTrigger'

Reply:
I see your point, but it sounds wrong like that. Perhaps some naming police with a major in the english language could come in a share their thought? :)